karaf/configs/com.cognifide.aet.runner.RunnerConfiguration.cfg

```diff
@@ -18,6 +18,6 @@
 
 ft=120
 mttl=300
-urlPackageSize=1
-maxMessagesInCollectorQueue=10
+urlPackageSize=5
```

Comment:
This number should be modified to 1, thanks for spotting that. I raised https://github.com/Cognifide/aet/issues/421 .